Examples/UIExplorer/UIExplorerApp.js

```diff
@@ -29,9 +30,30 @@ var UIExplorerApp = React.createClass({
   getInitialState: function() {
     return {
       openExternalExample: (null: ?React.Component),
+      tintColor: '#008888',
+      barTintColor: null,
+      titleTextColor: null,
     };
   },
 
+  _customNavColors: function() {
+    this.setState({
+      tintColor: '#FFFFFF',
+      barTintColor: '#4F6BA5',
+      titleTextColor: '#FFFFFF',
+    });
+    StatusBarIOS.setStyle(StatusBarIOS.Style['lightContent']);
```

Comment:
StatusBarIOS style switch removed, I changed the color of barTintColor to get better contrast (another blue, from React Native website :P)